### ISSUE TITLE
SM client requests get 403 with empty json body

### ISSUE THREAD

**Issue by ken-mills** [ghassani/surveymonkey-v3-api-php#28]
It appears that on 5/23, SM moved their platform to AWS Cloudfront which rejects requests that have an empty json body. My application uses SM webhooks to get notified that a new response is ready to download. We download survey responses many times per day. Our application started getting the 403s and returned false getData() calls around the same time. I submitted a support ticket to SM and got the attached response.

For some reason SM also change our access_token.

I changed Client.php, updated our access code and tested and all was ok. I will follow up with a pull request for review. Here is my attempt at a revised function.

```
private function createRequest($method, $uri, array $options = [], $body = null)
    {

        /* 
            Survey Monkey moved to CloudFront on 2020-05-23
            CloudFront issues 403 Forbidden with empty json body
        */    
        $ret = new Request($method, $uri, []);

        if (is_array($body)) {
            $bodyString = json_encode($body);
        }

        if (isset($options['query'])) {
            $uri = $ret->getUri()->withQuery(is_array($options['query']) ? http_build_query($options['query']) : $options['query']);
            return $ret->withUri($uri, true);
        }

        return $ret;
    }
```

![Annotation 2020-06-06 104503](https://user-images.githubusercontent.com/17298135/83947141-3e213580-a7e3-11ea-8ebf-86942f19183e.png)

**Comment by ghassani** [comment]
Commit dc3830ed75fc99a60fbdf507d74753b8a7d39081 has a fix for this issue in master branch now.

**Comment by ghassani** [comment]
Closing. Fixed with 1.1.1 Release
